JT/Example of README.md

```diff
@@ -0,0 +1,411 @@
+# Front-End Unit Test Workshop - Fixers
+
+### A. Error: Jest encountered an unexpected token
+Configure jest to compile ts files.
+
+<details>
+<summary>Solution</summary>
+
+- install **babel/preset-env**:
+- install **babel/preset-react**
```

Comment:
Missing colon at the end of the line. Would be nice to add everywhere where needed.

Reply:
this README file have purpose to show an example of how we can use markdown. At the moment we can skip the content.